app/ldap/server/loginHandler.js

```diff
@@ -46,23 +48,25 @@ Accounts.registerLoginHandler('ldap', function(loginRequest) {
 	const ldap = new LDAP();
 	let ldapUser;
 
+	const escapedUsername = ldapEscape.filter`${ loginRequest.username }`;
+
 	try {
 		ldap.connectSync();
-		const users = ldap.searchUsersSync(loginRequest.username);
+		const users = ldap.searchUsersSync(escapedUsername);
 
 		if (users.length !== 1) {
-			logger.info('Search returned', users.length, 'record(s) for', loginRequest.username);
+			logger.info('Search returned', users.length, 'record(s) for', escapedUsername);
 			throw new Error('User not Found');
 		}
 
 		if (ldap.authSync(users[0].dn, loginRequest.ldapPass) === true) {
-			if (ldap.isUserInGroup(loginRequest.username, users[0].dn)) {
+			if (ldap.isUserInGroup(escapedUsername, users[0].dn)) {
```

Comment:
The `users[0].dn` shouldn't be escaped as well? :thinking:

Reply:
@MarcosSpessatto not, since it's not a user input, it came from a previous search, so no risk to have undesired chars